src/test/java/com/flightstats/hub/dao/aws/ClusterContentServiceTest.java

```diff
@@ -0,0 +1,96 @@
+package com.flightstats.hub.dao.aws;
+
+import com.flightstats.hub.cluster.LastContentPath;
+import com.flightstats.hub.dao.ChannelService;
+import com.flightstats.hub.dao.ContentDao;
+import com.flightstats.hub.model.*;
+import com.flightstats.hub.util.HubUtils;
+import com.flightstats.hub.util.TimeUtil;
+import lombok.SneakyThrows;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClusterContentServiceTest {
+    @Mock private ClusterContentService ccs;
```

Comment:
I didn't know about this annotation. That's useful!

Reply:
Yeah, saves a little boilerplate on the mocking.